portfolio/src/main/webapp/blog.html

```diff
@@ -24,7 +24,7 @@ <h1 id="name">Xavier I. Luna Cabrera</h1>
           </p>
       </div>
 
-      <form name="entries" action='/data' method="POST">
+      <form name="entries" action='/data' method="POST" class="comnt-section">
         <label>Display number of comments </label>
```

Comment:
Is it possible to set the default comments to 5? and options as 5, 10, 20, 50? Matches more to what we see in blogs.

Reply:
yes, originally I left as 1 -4 comments because I though having more than that it would be a lot of comments, but since regular blogs have it I  will add it in.